### ISSUE TITLE
Shift biomass year by 1 in phase plot

### ISSUE THREAD

**Issue by iantaylor-NOAA** [pacific-hake/hake-assessment#443]
Mike wants final point to be fishing intensity from most recent year and biomass from Jan 1 on the following year. As discussed in SRG, assigning to 2019 milestone and closing.

**Comment by andrew-edwards** [comment]
For POP my co-author did this in the revision, but didn't change the axes labels. Super-confusing!

**Comment by andrew-edwards** [comment]
Suggest we not do this now. No-one else requested it.

**Comment by andrew-edwards** [comment]
Clarify in caption.

**Comment by aaronmberger-nwfsc** [comment]
Agree - I'm not all in on changing that figure.  Clarify in caption and mention whenever presenting this.

**Comment by cgrandin** [comment]
@aaronmberger-nwfsc did you actually change the caption? If not, this issue should stay open until it's done. We can leave this until we have the document building so as to ensure a lot of small bugs don't creep in.

**Comment by andrew-edwards** [comment]
I'm also going to look into `make.phase.plot()`. The credible interval (Figure h) implies about a 2.5% chance of biomass being <0.4B0, but the joint probability calc in the text comes out as 4.3% joint prob of this happening AND being above relative fishing intensity of 1. Which cannot be true.  
Manual calculation
```
sum(base.model$mcmc[[paste0("Bratio_", end.yr)]] < 0.4 ) / nrow(base.model$mcmc) * 100
```
gives 9.9% of being <0.4B0, so credible interval should dip further below 0.4 on the figure. I shall investigate (I think I wondered this last year also) ...

**Comment by andrew-edwards** [comment]
Okay, my last comment is solved by realising that the final plotted point is for 2019. i.e. B_2019 and relative-fish-intensity in 2019. (I'd assumed it showed B_2020 - biomass at start of 2020). 
So we have P(B_2019 < 0.4 B_0) = 2.65%, which agrees with the horizontal 95% credible interval just dipping below 0.4 B_0. So the caption needs clarifying (and the y-axis label) at the very least.

So, going to back to the original request in this issue. The x-axis is always B_t/B_0 (relative biomass at the start of year t, which we have estimated up to 2020). Then y-axis has two choices:
1. Relative fishing intensity in year t (as we have currently). But this does not show current biomass estimate.
2. Relative fishing intensity in year t-1 (as per Mike's request). So this would include the B_2020 estimate (which is the biomass at end of 2019 anyway). 

I prefer option 2 since it includes the current stock status. Looking online, I think people generally do either, or it's hard to tell because they don't label the axes properly. Er, like us :)  
It's not clear to me which is the standard. [Someone in one of the meetings suggested redefining biomass to be at the end of the year, but that's completely confusing and inconsistent with the whole assessment - however, caption could just point out that biomass at start of one year is the biomass at end of previous year.
So, any thoughts/comments? Vote for option 1 or option 2? [The code needs a bit of tidying up anyway].

**Comment by cgrandin** [comment]
I think option 2 makes more sense as the biomass at the beginning of a year should be with the catch from the year prior having been removed..

**Comment by aaronmberger-nwfsc** [comment]
Option 2 makes sense to me as well, which if I'm understanding means leaving the plot the way it is and just clarifying what points refer to.  I'm fine with just being more explicit in the caption what years the x-axis and y-axis refer to, however we have to then also clarify what year labels mean in the figure itself because the year label will directly refer to only one of the axes.

**Comment by andrew-edwards** [comment]
Good point about the label. No, plot will change since y-axis is currently year t, same as x-axis. I'll work it now...

**Comment by andrew-edwards** [comment]
Fixed in b846ab7. More fiddly than expected, makes more sense now though.

**Comment by cgrandin** [comment]
@andrew-edwards Please check text in Management Performance section, (search for "pending issue
443 resolution"). The joint probability may need updating and I will leave this to you to do and close this.
